core/src/test/java/io/questdb/test/griffin/ExplainPlanTest.java

```diff
@@ -785,13 +785,12 @@ public void testDistinctTsWithLimit2() throws Exception {
         assertPlan(
                 "create table di (x int, y long, ts timestamp) timestamp(ts)",
                 "select distinct ts from di order by 1 desc limit 10",
```

Comment:
Would be great to add one more test involving other columns in the select clause, not only the designated timestamp. And one more with a few columns in order by.

Reply:
I added a test with multiple columns in the select clause: a999987126f85181522968fca52c19d460e6c9fd. The second part (multiple columns in the ORDER BY clause) I'll address as a separate issue/PR, as we discussed on Slack.